GUI/QtQuick/qml/CreateWallet/ImportWallet.qml

```diff
@@ -315,7 +316,7 @@ ColumnLayout  {
 
         visible: !grid.isValid
 
-        text: "Not a valid words"
+        text:  "Words are not valid"
```

Comment:
The text is still not translatable (`qsTr`)

Reply:
ready